src/components/RegistrationList.tsx

```diff
@@ -106,7 +106,14 @@ export const RegistrationListItemContent = ({
   return (
     <Box sx={{ display: 'flex', width: '100%', gap: '1rem' }}>
       <ListItemText disableTypography data-testid={dataTestId.startPage.searchResultItem}>
-        <Box sx={{ display: 'flex', alignItems: 'center', gap: { xs: '1rem', sm: '2rem' }, marginBottom: '0.5rem' }}>
+        <Box
+          sx={{
+            display: 'flex',
+            alignItems: 'center',
+            gap: { xs: '1rem', lg: '2rem' },
```

Comment:
Kan sikkert diskuteres om vi faktisk trenger annen gap på medium enn large, eller om 1rem kanskje alltid er nok
```suggestion
            gap: '1rem',
```

Reply:
Kanskje skulle gap her vært mindre på row også feks 0.5rem, for tipper 2rem på både column og row blir noksåluftig når det er en kladd nå

Reply:
Ettersom det brukes `wrap` og ikke `flexDirection` blir gapen brukt dynamisk. Synes `1rem` funker over hele fjøla, så går for det.